packages/lab/src/dropdown-next/DropdownNext.css

```diff
@@ -0,0 +1,111 @@
+  padding-left: var(--saltDropdownNext-paddingLeft, var(--dropdownNext-padding));
+  padding-right: var(--saltDropdownNext-paddingRight, var(--dropdownNext-padding));
+  position: relative;
+  border: none;
+  border-bottom: var(--dropdownNext-borderWidth) var(--dropdownNext-borderStyle) var(--dropdownNext-borderColor);
+  justify-content: space-between;
+}
+
+.saltDropdownNext-button:hover {
+  --dropdownNext-borderColor: var(--salt-editable-borderColor-hover);
+
+  cursor: var(--salt-selectable-cursor-hover);
+}
+
+.saltDropdownNext-button:focus,
+.saltDropdownNext-button:active {
+  --dropdownNext-borderColor: var(--salt-editable-borderColor-active);
+  --dropdownNext-borderWidth: var(--salt-editable-borderWidth-active);
+
+  outline: var(--saltDropdownNext-outline, var(--salt-focused-outlineWidth) var(--salt-focused-outlineStyle) var(--dropdownNext-outlineColor));
+}
+
+.saltDropdownNext-buttonText {
+  text-align: left;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  padding-right: var(--salt-size-adornmentGap);
+}
+
+.saltDropdownNext-list {
+  --saltDropdownNext-itemCount: 5;
+
+  border-color: var(--salt-selectable-borderColor-selected);
+  max-height: calc((var(--salt-size-base) + var(--salt-spacing-100)) * var(--saltDropdownNext-itemCount));
+}
+
+/* Styling applied to dropdown button if `disabled={true}` */
+.saltDropdownNext-disabled,
+.saltDropdownNext-disabled:hover,
+.saltDropdownNext-disabled:active {
+  --dropdownNext-borderColor: var(--salt-editable-borderColor-disabled);
+  --dropdownNext-borderStyle: var(--salt-editable-borderStyle-disabled);
+  --dropdownNext-borderWidth: var(--salt-size-border);
+
+  cursor: var(--salt-selectable-cursor-disabled);
+  background: var(--dropdownNext-background-disabled);
+  color: var(--saltDropdownNext-color-disabled, var(--salt-text-primary-foreground-disabled));
+  outline: 0;
+}
+
+/* Style applied to dropdown button if `readOnly={true}` */
+.saltDropdownNext-button.saltDropdownNext-readOnly,
+.saltDropdownNext-button.saltDropdownNext-readOnly:hover,
+.saltDropdownNext-button.saltDropdownNext-readOnly:active {
+  --dropdownNext-borderColor: var(--salt-editable-borderColor-readonly);
+  --dropdownNext-borderStyle: var(--salt-editable-borderStyle-readonly);
+  --dropdownNext-borderWidth: var(--salt-size-border);
+
+  cursor: var(--salt-editable-cursor-readonly);
+  background: var(--dropdownNext-background-readonly);
+  padding-left: 0;
+  outline: 0;
+}
+
+/* Styling applied to dropdown button icon if `disabled={true}` or `readOnly={true}` */
+.saltDropdownNext-icon.saltDropdownNext-disabled,
+.saltDropdownNext-icon.saltDropdownNext-disabled:hover,
+.saltDropdownNext-icon.saltDropdownNext-disabled:active,
+.saltDropdownNext-icon.saltDropdownNext-readOnly,
+.saltDropdownNext-icon.saltDropdownNext-readOnly:hover,
+.saltDropdownNext-icon.saltDropdownNext-readOnly:active {
+  --icon-color: var(--salt-text-primary-foreground-disabled);
```

Comment:
shouldnt define --icon variable in dropdown, plus not redefined, just use characteristic directly